client/signup/steps/woocommerce-install/components/support-card/index.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { ReactElement } from 'react';
+import { useSelector } from 'react-redux';
+import { getSiteDomain } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+const SupportLinkContainer = styled.p`
+	@media ( max-width: 320px ) {
+		margin-top: 0px;
+		width: 100%;
+	}
+`;
+
+const SupportLinkStyle = styled.a`
+	/* Gray / Gray 100 - have to find the var value for this color */
```

Comment:
I should've included removing this line in my suggestion:

```suggestion
```

Reply:
sorry forgot to remove it. Thanks.